refactor_me.py

```diff
@@ -23,4 +22,4 @@ def print_expenses(expenses):
 if __name__ == '__main__':
     # TODO(dmu) HIGH: Use static fixtures and dynamic fixture framework instead
     test_expenses = (Expense('food', 4), Expense('food', 3), Expense('car', 3), Expense('dog', 1))
-    print_expenses(sum_expenses(test_expenses, 2))
+    print_expenses(sum_expenses(expenses=test_expenses, min_amount=2))
```

Comment:
`min_amount=2` is a good point because it is keyword argument, but I would not go with `expenses=test_expenses`, because it is a positional argument.